docs/scripts/arxiv_references.py

```diff
@@ -176,9 +176,9 @@ def search_code_for_arxiv_references(code_dir: Path) -> dict[str, set[str]]:
             else:
                 module_name_and_member_reduced.add(module_name_and_member)
         if module_name_and_member_reduced:
-            arxiv_id2module_name_and_members_reduced[arxiv_id] = (
-                module_name_and_member_reduced
-            )
+            arxiv_id2module_name_and_members_reduced[
```

Comment:
Was this just a linting change?

Reply:
Yes. Came from running `make format` or one of the other commands of theirs